.github/workflows/test-function-kysely-tailordb-codegen.yaml

```diff
@@ -12,9 +12,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
-      - uses: actions/setup-node@a0853c24544627f65ddf259abe73b1d18a591444 # v5.0.0
+      - uses: actions/setup-node@2028fbc5c25fe9cf00d9f06a71cc4710d4507903 # v6.0.0
```

Comment:
With v5, it seems to try to enable pnpm caching by default and throws an error, so I've updated to v6.
I think this went unnoticed because the workflow probably hasn't been run once since updating to v5...
https://github.com/actions/setup-node/issues/1357